src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -658,6 +658,42 @@ namespace DataHeatBalance {
         std::vector<ExteriorEnergyUse::ExteriorFuelUsage> otherEquipFuelTypeNums; // List of fuel types used by other equipment in this zone
         std::vector<std::string> otherEquipFuelTypeNames;                         // List of fuel types used by other equipment in this zone
 
+        // People Thermal Resilience Related
+        Real64 ColdStressTempThresh = 15.56;
```

Comment:
@mjwitte I had suggested moving some of the standalone Array1D objects into plain scalars on the zone data structure.  Doing this will avoid having separate arrays, each with their own heap allocation, bookkeeping and flags to track what has been allocated and what hasn't...when all that is needed is a flag on the zone data structure.  Is your concern about the list of variables on ZoneData getting too large and hard to track?  Would it be more appealing if all these were collected into a small struct on its own, with a single member on the ZoneData structure?  Like:

```
struct ZoneResilience {
  Real64 someThreshold = 0.0;
  Real64 otherValue = 63.1;
  //...
}

struct ZoneData {
  // ... lots of zone data
  ZoneResilience resilience;
}

// then when you access it in the code, it would be like:
auto &thisZone = ZoneData(ZoneNum):
x = y + thisZone.resilience.someThreshold;
```

That seems like a fairly easy and nice change to make; what do you think?

Reply:
Hi @Myoldmopar, I discussed this with @mjwitte yesterday. The concern is zone data will get too large and potentially necessarily hogging along a whole bunch of unused data fields when resilience reports are not requested. A separate struct holding only resilience data fields will potentially help with this situation. Or we were also thinking we could even break the resilience table-related struct into subgroups and make them into separate structs if the reliance-related data fields alone are too many to be held in one struct. I was working on investigating the shading feature. I hope to get back to this feature and implement the rearrangement during the weekend.